From 9a8de07954c120e16ae27a4c2a44aa67d7609ab1 Mon Sep 17 00:00:00 2001
From: Tom Powell <tom@powell.io>
Date: Thu, 24 May 2018 20:53:18 -0700
Subject: [PATCH 205/214] Privacy policy: clarify Gerrit information
 disclosure.

Gerrit discloses email addresses, names, and usernames as provided
by either users or the oauth provider (currently Google). Clarify
this, along with a waiver to request removal of previously-committed
information (as rewriting history to remove personal data is both
destructive and counter-productive).

Change-Id: If6408c2ea6b3314d333771bb23b2d2f0adbe0815
---
 legal.md | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/legal.md b/legal.md
index 1d0e1e4..00d8a3d 100644
--- a/legal.md
+++ b/legal.md
@@ -55,6 +55,11 @@ We do not share your information with companies, organizations, or individuals o
 - With your consent (Submission of bug reports to our bug tracking platform is considered consent.)
 - When legally required.
 
+When using our code review platform (Gerrit), you agree that:
+
+- During account creation, we collect your name, username, and email from a third party. This data is immediately made public.
+- Your name and email address will become embedded and part of the code, which will be publicly available. You understand that the removal of this information would be impermissibly destructive to the project and the interests of all who contribute, utilize, and benefit from it. You waive any right to request erasure, removal, or rectification of this information under any applicable privacy or other law and understand that providing this information is required to contribute to the project.
+
 We will share the information we have with entities outside of LineageOS if we have a good faith belief that access, use, preservation, or disclosure of the information is necessary to:
 
  - meet any applicable law, regulation, legal process or enforceable governmental request.
-- 
2.17.1

